src/linker/utilities/spark.smk

```diff
@@ -117,6 +117,9 @@ rule start_spark_worker:
     params:
         terminate_file_name=rules.terminate_spark.output,
         user=os.environ["USER"],
+        cores=1,
```

Comment:
Isn't this still effectively hardcoding resource requests? Or is the idea that this is only used to spin up a cluster and so 1GB and 1 core should always be enough?

Reply:
This is basically a 'default' value for the params-- when we write the main snakefile above with `use rule start_spark_worker from spark_cluster with:` we override the values of these parameters